components/xx-rootmail.yaml

```diff
@@ -0,0 +1,329 @@
+            }
+          ]
+        }
+
+  HostedZone:
+    Type: AWS::Route53::HostedZone
+    Properties:
+      Name: !Sub ${Subdomain}.${Domain}
+      HostedZoneConfig:
+        Comment: Created by superwerker
+
+  HostedZoneDelegationHealthCheck:
+    DependsOn: HostedZoneDelegationHealthCheckRecord
+    Type: AWS::Route53::HealthCheck
+    Properties:
+      HealthCheckConfig:
+        FailureThreshold: 3
+        FullyQualifiedDomainName: !Sub ${Subdomain}.${Domain}
+        RequestInterval: 30
+        Type: HTTP
+      HealthCheckTags:
+        - Key: Name
+          Value: RootEmailsHostedZoneDelegationHealthCheck
+
+  HostedZoneDelegationHealthCheckRecord:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      AliasTarget:
+        DNSName: !Sub
+          - "${DomainBucket}.${Endpoint}"
+          - Endpoint: !FindInMap
+            - RegionMap
+            - !Ref "AWS::Region"
+            - Endpoint
+        HostedZoneId: !FindInMap
+          - RegionMap
+          - !Ref "AWS::Region"
+          - HostedZoneID
+      HostedZoneId: !Ref HostedZone
+      Name: !Sub ${Subdomain}.${Domain}
+      Type: A
+
+  HostedZoneDKIMAndVerificationRecords:
+    Type: AWS::CloudFormation::CustomResource
+    Properties:
+      ServiceToken: !GetAtt HostedZoneDKIMAndVerificationRecordsCustomerResource.Arn
+      Domain: !Sub ${Subdomain}.${Domain}
+
+  HostedZoneDKIMAndVerificationRecordsCustomerResource:
+    Type: AWS::Serverless::Function
+    Properties:
+      Timeout: 200
+      Handler: index.handler
+      Runtime: python3.7
+      Policies:
+        - Version: 2012-10-17
+          Statement:
+            - Effect: Allow
+              Action: # TODO: least privilege
+                - ses:*
+              Resource: "*"
+      InlineCode: |
+        import boto3
+        import cfnresponse
+
+        ses = boto3.client("ses")
+
+        CREATE = 'Create'
+        DELETE = 'Delete'
+        UPDATE = 'Update'
+
+        def exception_handling(function):
+            def catch(event, context):
+                try:
+                    function(event, context)
+                except Exception as e:
+                    print(e)
+                    print(event)
+                    cfnresponse.send(event, context, cfnresponse.FAILED, {})
+
+            return catch
+
+        @exception_handling
+        def handler(event, context):
+            RequestType = event["RequestType"]
+            Properties = event["ResourceProperties"]
+            LogicalResourceId = event["LogicalResourceId"]
+            PhysicalResourceId = event.get("PhysicalResourceId")
+            Domain = Properties["Domain"]
+
+            print('RequestType: {}'.format(RequestType))
+            print('PhysicalResourceId: {}'.format(PhysicalResourceId))
+            print('LogicalResourceId: {}'.format(LogicalResourceId))
+
+            id = PhysicalResourceId
+
+            data = {}
+
+            if RequestType == CREATE:
```

Comment:
how about DELETE actions?

Reply:
There is nothing created here, just outputs populated.